browse/static/css/arXiv.css

```diff
@@ -54,7 +54,7 @@ img {
 /* Following command not used by abs samples, but present in arXiv.css*/
 em {
   color: #c00;
-  font-weight: bold;
+  /*font-weight: bold;*/
```

Comment:
Is there a reason this is commented out? Changing em might affect other pages.

Reply:
Ah, I did that in order to make the (withdrawn) to the right of the version number no be bold. I will make a new style for that. I could also make the (withdrawn) not red if you'd prefer.

Reply:
![plain](https://user-images.githubusercontent.com/56355455/235377497-1fe2b1f9-29a0-4330-8948-6156eba889d8.PNG)
![bold](https://user-images.githubusercontent.com/56355455/235377504-2cf0f8bd-fdb5-4134-b357-31b77e93dd62.PNG)

Reply:
Here is what plain and bold (withdrawn) looks like. I almost prefer the bold. Let me know what you think.

Reply:
I think the bold is appropriate.